src/main/java/training/endava/playground/generics/types/LimitedContainer.java

```diff
@@ -0,0 +1,29 @@
+package training.endava.playground.generics.types;
+import java.util.*;
+public class LimitedContainer<T, N extends Number> {
+private List<T> list=new ArrayList<>();
+private N restriction;
+
+public LimitedContainer(N restriction){
+    this.restriction=restriction;
+
+}
+
+public void copyInContainer(List<T> list){
+    this.list=list;
+
+}
+public List<T> getContainer(){
+
+    return this.list;
+}
+
+public void addElemInContainer(T elem){
+
+    if(this.list.size()< this.restriction.intValue()){
+        this.list.add(elem);
+    }
+    else System.out.println("Max capacity");
```

Comment:
please indent and use brackets
what happens if you want to add a new line in the else block?